### DISCUSSION TITLE
Description for menu items in a menu

### DISCUSSION THREAD

**Discussion by grbba** [daniele77/cli#95] [Ideas / Feature requests]
I start that as a separate discussion item for more clarity.

```
Commands available:
 - help
        This help message
 - exit
        Quit the session
 - config
        Shows the configuration items for the current session
 - color
        Enable colors in the cli
 - nocolor
        Disable colors in the cli
 - build
        (menu)
 - cs
        (menu)
```

the last two menu items relate to submenus ( or 'modes' as for the purpose of the app ). There is only an indication that these things are a menu but not their purpose. So can i have something like

```
 - build Switches to build mode for <purpose_text>
        (menu)
 - cs Switches to commandstation mode for <purpose_text>
        (menu)
```
or 

```
 - build 
        Switches to build mode for <purpose_text> (menu)
 - cs 
       Switches to commandstation mode for <purpose_text>  (menu) 
```

Thanks a lot.

**Comment by daniele77** [comment]
I think this is a good idea.
I just opened a new issue for the enhancement (#96).

**Reply by grbba** [reply]
Cool that you added this one to the issues.
I have another one ..
Actually the name you give in the menu construction is the command to change to the submenu but also the label for the prompt. It would be nice if you could have a different prompt though. i'd like to keep the command short eg 'cs' but the prompt to be 'CommandStation' which gives more contextual info to the user. 

//

**Reply by daniele77** [reply]
This is a good idea, too. Please submit a new issue for this (maybe you can link this discussion to it).
Thank you

**Reply by grbba** [reply]
I have added that to the issue list 👍 

//
